src/ca11y/index.js

```diff
@@ -4,32 +4,93 @@ import zeroPad  from './lib/zeroPad'
 import keys     from './lib/keys'
```

Comment:
:hocho: the `zeroPad` import, but leave it as a file available to commonjs consumers of this lib.